lib/Extension/Command/IndexBuildCommand.php

```diff
@@ -121,4 +121,14 @@ private function watch(OutputInterface $output): void
             }
         });
     }
+
+    /**
+     * @see https://www.php.net/manual/en/function.memory-get-usage.php#96280
+     */
+    private function formatMemory(int $memoryInBytes): string
+    {
+        $unit = ['B', 'KiB', 'MiB', 'GiB'];
+
+        return @round($memoryInBytes / pow(1024, ($i = floor(log($memoryInBytes, 1024)))), 2).' '.$unit[$i];
```

Comment:
could we use `sprintf('%s %s')` and remove the inline assign, why do we ignore the error from round?

Reply:
this "logic" would also need to be used by the language server indexer service.

does it add any value? we currently report this in megabytes - bytes and kilobytes don't matter - and gigabytes should be rare and would be represented as `1,000mb` which is fine

Reply:
ok, i guess this is for the final memory usage report - i'd still be inclined to leave it as is.

Reply:
I simply copy/paste from the php docucmentation, link in the comment.
There is no value it's just prettier on the screen and easier to read.
For the `@` I don't know, maybe to protect against a division by zero or whatever. I don't really care about this like that, it's sugar it's not a code to maintain or anything so I admit I didn't look past the fact that it worked.
I will try to make it cleaner.

Reply:
well - it's code to maintain if it fails for whatever reason (copy / paste from PHP.net has got me in trouble before now)

Reply:
> ok, i guess this is for the final memory usage report - i'd still be inclined to leave it as is.

So I revert the related commit ?